CONTRIBUTING.md

```diff
@@ -66,3 +66,45 @@ If a Crowdin translation like this `You have %num% left in your wallet` - would
 ```
 TranslateString(675, `You have ${cakeBalance} left in your wallet`, { num: cakeBalance })
 ```
+
+<a name="bugs"></a>
```

Comment:
@guifeliper `name` isn't a valid <a> attribute, and why is this an empty anchor?

Reply:
hmm I have followed a template, my first thought was to create a link, but actually does not make sense, you are right.